AVInput/AVInput.config

```diff
@@ -1,3 +1,9 @@
 set (autostart false)
 set (preconditions Platform)
 set (callsign org.rdk.AVInput)
+
+if(PLUGIN_AVINPUT_STARTUPORDER)
+map()
```

Comment:
Why do we need map() here? isn't it enough to just set env variable? like autostart above

Reply:
The reference patch had similar configuration; didn't wanted to deviate much from it.
But it seems not required.

Reply:
I think it'd be more readable if you drop not needed map()/end().

Reply:
Updated the changes after a local build verification. Will put the same for sprint verification also.